pytest_reportportal/plugin.py

```diff
@@ -60,9 +60,11 @@ def pytest_sessionstart(session):
             retries=int(session.config.getini('retries')),
         )
 
+        tags = [{'value': tag} for tag in
+                session.config.getini('rp_launch_tags')]
```

Comment:
As we implemented a new entity in RP5 - attributes, I believe there should be another property: `rp_launch_attributes`. I can try to implement it by myself if you have lack of time.
Java example: 
`rp.attributes=key1:value1;key2:value2`